crates/ide-db/src/path_transform.rs

```diff
@@ -11,7 +11,9 @@ use syntax::{
 
 #[derive(Default)]
 struct AstSubsts {
-    types: Vec<ast::TypeArg>,
+    // ast::TypeArgs stands in fact for both type and const params
+    // as consts declared elsewhere look just like type params.
```

Comment:
not sure if I should report it

Reply:
This is expected, the parser cannot differentiate that, so for paths, type args might be const references but that cannot be known until the path is resolved semantically